src/locale/misc/tasks/en.yml

```diff
@@ -39,3 +39,8 @@ publishButton:
         missingAssignees: Missing assignees
         missingFieldsAndAssignees: Missing required fields and assignees
         alreadyPublished: Already Published
+taskAssigneesList:
+    title: Assigned to {numPeople} {numPeople, plural, one {person} other {people}}
```

Comment:
Hint: This can also be written as `Assigned to {numPeople, plural, one {# person} other {# people}}`, i.e. using `#` to interpolate the number into the pluralized version of the string instead of doing it separately outside of pluralization with `{numPeople}`.

This can be useful to do things like rendering "one person" instead of "1 person", but also for languages where the numeral might be affected by the pluralization.

Reply:
I'm gonna do that! ^